proc/proc_linux.go

```diff
@@ -71,9 +71,15 @@ func (dbp *Process) Kill() (err error) {
 	if err != nil {
 		return errors.New("could not deliver signal " + err.Error())
 	}
-	_, _, err = wait(-1, dbp.Pid, 0)
```

Comment:
What about replacing `-1` with `dbp.Pid` to explicitly wait for the process pid, instead of iterating?